cc/toolchains/llvm/swift_custom_features.bzl

```diff
@@ -88,6 +88,10 @@ swift_no_default_warnings = feature(
                 "-Wno-fortify-source",
                 "-Wno-absolute-value",
                 "-Wno-format",
+                "-Wno-deprecated-declarations",
```

Comment:
A few default compiler warnings I missed in the last PR.

Again these are only silenced for third party code (curl, suiteparse, hdf5, etc..)